subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -97,6 +98,13 @@ struct notification_attrs {
 	uint8_t flags;
 };
 
+#ifdef CONFIG_LWM2M_VERSION_1_1
+#define LWM2M_ENGINE_MAX_OBSERVER_PATH CONFIG_LWM2M_ENGINE_MAX_OBSERVER * 3
```

Comment:
Would be good to document what is the reason for tripling the value. Another things is that should this be configurable though Kconfig instead.

Reply:
good point it is difficult to estimate purpose. Now it allocate CONFIG_LWM2M_ENGINE_MAX_OBSERVER *3 which means that client can allocate CONFIG_LWM2M_ENGINE_MAX_OBSERVER time composite operation with 3 path.